packages/demo/src/common/Collection/signals/init.js

```diff
@@ -0,0 +1,33 @@
+import {input, merge, set, state, when} from 'cerebral/operators'
+import {onChildAdded, onChildChanged, onChildRemoved, value} from 'cerebral-provider-firebase'
+import paths from '../paths'
+
+export default function init (moduleName, initState = {}) {
+  const {collectionPath, dynamicPaths, errorPath} = paths(moduleName)
+  return [
+    // prepare remote and local path
+    ...dynamicPaths,
+    // start listening
+    onChildAdded(
+      input`remoteCollectionPath`, `${moduleName}.updated`),
```

Comment:
I think this should go inside the success of the value call so that we do not risk missing items.